src/sql/workbench/browser/modelComponents/text.component.ts

```diff
@@ -18,15 +18,15 @@ import { TitledComponent } from 'sql/workbench/browser/modelComponents/titledCom
 @Component({
 	selector: 'modelview-text',
 	template: `
-		<div *ngIf="showDiv;else noDiv" style="display:flex;flex-flow:row;align-items:center;" [style.width]="getWidth()">
-		<p [innerHTML]="getValue()" [title]="title" [ngStyle]="this.CSSStyles" (click)="onClick()"></p>
+	<div *ngIf="showDiv;else noDiv" style="display:flex;flex-flow:row;align-items:center;" [style.width]="getWidth()">
```

Comment:
display property is not applied to this element.

Reply:
Correct - as mentioned in the description this was trickier because it already has the display property set to flex and I didn't have any super obvious suggestions for how to fix this (possibly have the flex display be in another nested div). If you have any suggestions let me know and I can go back and fix this but for the time being I'm just going to assume that if someone needs this in the future they'll add that functionality in themselves.